sup3r/preprocessing/data_handling/dual_data_handling.py

```diff
@@ -183,10 +183,9 @@ def means(self):
         dict
         """
 
-        if self.lr_dh.data is None or self.hr_dh.data is None:
-            msg = ('Low-res or High-res DataHandler object has '
-                   'DataHandler.data=None! Try initializing DualDataHandler '
-                   'with load_cached=True')
+        if self.hr_dh.data is None:
+            msg = ('High-res DataHandler object has DataHandler.data=None! '
```

Comment:
Maybe this should say try initializing the high-res handler with load_cached=True.

Reply:
updated